lib/models/project.js

```diff
@@ -529,7 +529,8 @@ Project.prototype.findAddonByName = function(name) {
   }
 
   return find(this.addons, function(addon) {
-    return name.indexOf(addon.name) > -1 || (addon.pkg && name.indexOf(addon.pkg.name) > -1);
+    return (name.slice(addon.name && -addon.name.length) === addon.name) ||
```

Comment:
`ember-cli-qunit` has a `addon.name`, and even if it didn't its `pkg.name` isn't `ember-cli/ember-cli-qunit`.